roles/confluent.variables/defaults/main.yml

```diff
@@ -1101,6 +1106,314 @@ control_center_health_check_user: "{{control_center_ldap_user}}"
+kafka_connect_replicator_group: "{{kafka_connect_replicator_default_group}}"
+
+### Variable to define bootstrap servers for Kafka Connect Replicator.  Mandatory for Kafka Connect Replicator setup.
+kafka_connect_replicator_bootstrap_servers: localhost:9092
+
+### Use to set custom Kafka Connect Replicator properties. This variable is a dictionary. Put values true/false in quotation marks to perserve case.
+kafka_connect_replicator_custom_properties: {}
+
+### Set this variable to customize the Cluster ID used by Kafka Connect Replicator.
+kafka_connect_replicator_cluster_id: replicator
+
+### Set this variable to customize the offset starting point for Kafka Connect Replicator.
+kafka_connect_replicator_offset_start: consumer
+
+### Set this variable to customize the topic that Kafka Connect Replicator stores it's offsets in.
+kafka_connect_replicator_offsets_topic: connect-offsets
+
+### Set this variable to customize the topic where Kafka Connect Replicator stores it's status.
+kafka_connect_replicator_status_topic: connect-status
+
+### Set this variable to customize the topic where Kafka Connect Replicator stores it's configuration.
+kafka_connect_replicator_storage_topic: connect-configs
+
+### Set this variable with a comma seperated list of Topics for Kafka Connect Replicator to replicate from.  This is a mandatory variable.
+kafka_connect_replicator_white_list: ""
+
+### Sets if Topics are auto created on the destintation cluster by Kafka Connect Replicator.
+kafka_connect_replicator_topic_auto_create: true
+
+### Boolean to enable health checks on Kafka Connect Replicator.
+kafka_connect_replicator_health_checks_enabled: true
+
+### User for authenticated Kafka Connect Replicator Health Check. Set if using customized security like Basic Auth.
+kafka_connect_replicator_ldap_user: connect
+
+### Password for authenticated Connect Replicator Health Check. Set if using customized security like Basic Auth.
+kafka_connect_replicator_ldap_password: password
+
+### Port Rest API exposed over.
+kafka_connect_replicator_port: 8083
+
+### Sets Kafka Connect Replicator to enable monitoring intercepotors for monitoring in Control Center.
+kafka_connect_replicator_monitoring_interceptors_enabled: true
+
+### User for authenticated Connect Health Check. Set if using customized security like Basic Auth.
+kafka_connect_replicator_health_check_user: "{{kafka_connect_replicator_ldap_user}}"
+
+### Password for authenticated Connect Health Check. Set if using customized security like Basic Auth.
+kafka_connect_replicator_health_check_password: "{{kafka_connect_replicator_ldap_password}}"
+
+kafka_connect_replicator_http_protocol: "{% if kafka_connect_replicator_monitoring_interceptor_listener ['ssl_enabled'] | default(ssl_enabled) | bool%}https{%else%}http{%endif%}"
+
+kafka_connect_replicator_jolokia_ssl_enabled: "{{kafka_connect_replicator_monitoring_interceptor_listener ['ssl_enabled'] | default(ssl_enabled) | bool}}"
```

Comment:
is there a variable `kafka_connect_replicator_ssl_enabled`? which should define if the rest endpoint has ssl encryption?
This code right now looks at weather the monitoring kafka listener has ssl enabled